hooks/boot/01-kafo-hook-extensions.rb

```diff
@@ -164,6 +164,11 @@ def execute_as!(user, command, do_say = true, do_log = true, extra_env = {})
     execute!(runuser_command, do_say, do_log, extra_env)
   end
 
+  def execute_as_formatted!(user, command, do_say = true, do_log = true, extra_env = {})
```

Comment:
I like using https://github.com/theforeman/kafo/commit/c463e3f38f5477a94460f1b7dbd8d8eedd87801d to greatly reduce the amount of escaping needed, but have some questions about the implementation.

It's called `execute_as_formatted!` and `!` implies it raises an exception if it fails (copied from Rails, see https://stackoverflow.com/questions/612189/why-are-exclamation-marks-used-in-ruby-methods for more discussion on the convention). `execute_command` doesn't, so that's a mismatch.

On the name itself, I think formatted could be confusing because I don't see any formatting. Naming is hard, but any thoughts on `execute_as_user_verbatim`?

Reply:
How does `execute_preformatted_as` sound?